src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -220,7 +221,7 @@ def step(
         model_output: torch.FloatTensor,
         timestep: int,
         sample: torch.FloatTensor,
-        predict_epsilon=True,
```

Comment:
So far `predict_epsilon` was never used no, so I guess we could deprecate this functionality easily here. 
`prediction_type` seems like a reasonable name to me - what do you think @anton-l ?

Reply:
It's used in the RL colab.

Reply:
Where can I find these RL colabs? Perhaps, I can make changes in them to use the current implementation with `prediction_type`?